frappe/email/queue.py

```diff
@@ -403,8 +403,8 @@ def send_one(email, smtpserver=None, auto_commit=True, now=False, from_test=Fals
 
 	try:
 		if not frappe.flags.in_test:
-			if not smtpserver: smtpserver = SMTPServer()
-			smtpserver.setup_email_account(email.reference_doctype, sender=email.sender)
+			if not smtpserver: smtpserver = SMTPServer(append_to=email.reference_doctype, sender=email.sender)
```

Comment:
```suggestion
			if not smtpserver: smtpserver = SMTPServer()
```
No need to pass any arguments to SMTPServer while creating object, as you are anyway passing sender to `smtpserver.setup_email_account`